src/Workspaces/VisualBasic/Portable/FindSymbols/VisualBasicDeclaredSymbolInfoFactoryService.vb

```diff
@@ -127,171 +144,198 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.FindSymbols
                typeDecl.Members.Any() AndAlso
                typeDecl.Members.All(Function(m) TypeOf m Is TypeBlockSyntax) Then
 
-                declaredSymbolInfo = Nothing
-                Return False
+                Return
+            End If
+
+            If node.Kind() = SyntaxKind.PropertyBlock Then
+                node = DirectCast(node, PropertyBlockSyntax).PropertyStatement
+            ElseIf node.Kind() = SyntaxKind.EventBlock Then
+                node = DirectCast(node, EventBlockSyntax).EventStatement
+            ElseIf TypeOf node Is MethodBlockBaseSyntax Then
+                node = DirectCast(node, MethodBlockBaseSyntax).BlockStatement
             End If
```

Comment:
the additional of MethodBlockBaseSyntax here is new.  We effectively did not support abstract VB method before in navto.  i fixed that as part of this change.